test/Benchmarks/Logs/LogBenchmarks.cs

```diff
@@ -19,6 +19,25 @@
 using OpenTelemetry;
 using OpenTelemetry.Logs;
 
+/*
+// * Summary *
+
+BenchmarkDotNet = v0.13.1, OS = Windows 10.0.22000
+Intel Core i7-8650U CPU 1.90GHz (Kaby Lake R), 1 CPU, 8 logical and 4 physical cores
+.NET SDK=6.0.101
+  [Host]     : .NET 6.0.1(6.0.121.56705), X64 RyuJIT
+  DefaultJob : .NET 6.0.1(6.0.121.56705), X64 RyuJIT
+
+
+|                                 Method |     Mean |    Error |   StdDev |   Median |  Gen 0 | Allocated |
+|--------------------------------------- |---------:|---------:|---------:|---------:|-------:|----------:|
+|                             NoListener | 135.6 ns | 10.90 ns | 32.15 ns | 125.7 ns | 0.0153 |      64 B |
+|                           OneProcessor | 251.9 ns | 16.29 ns | 46.73 ns | 241.9 ns | 0.0553 |     232 B |
```

Comment:
Is this much allocation expected?

Reply:
At least `>= sizeof(LogRecord)` until we could reuse the records. The good thing is that these records are short-lived comparing to Activities which could live much longer.